src/main.cpp

```diff
@@ -88,7 +85,8 @@ main(int argc, char *argv[])
                 return 2;
             }
 
-            rpms << info.absoluteFilePath();
+            rpms << info.absoluteFilePath().prepend("file://");
```

Comment:
QUrl::fromLocalFile()? Commonly should result the same, but then we have url percent encoding etc. Rpms likely having safer names than most, but generally I'd avoid this kind of url conversions.